src/compas_eve/mqtt/mqtt_paho.py

```diff
@@ -103,8 +106,9 @@ def subscribe(self, topic, callback):
         subscribe_id = "{}:{}".format(event_key, id(callback))
 
         def _local_callback(msg):
-            msg = topic._message_from_json(msg.payload.decode())
-            callback(msg)
+            decoded_data = self.codec.decode(msg.payload.decode())
+            message_obj = topic.message_type.parse(decoded_data)
+            callback(message_obj)
```

Comment:
Shouldn't the `message_type.parse` be inside the codec instances, to be symmetric with the fact that it also handles conversion directly from `Message` instance in the encoding step?